CosmoSpaceShip/CosmoSpaceShip/Src/Game.cpp

```diff
@@ -151,7 +167,7 @@ void Game::handleEvents()
 	switch (event.type)
 	{
 	case SDL_QUIT :
-		isRunning = false;
+	//	isRunning = false;
 		break;
 	default:
 		break;
```

Comment:
refactor player handing in update function!!!!!!!!!!!!!!!!!!
And every other place too

Reply:
Done.